tool/tctl/common/alert_command.go

```diff
@@ -181,7 +181,11 @@ func (c *AlertCommand) List(ctx context.Context, client auth.ClientI) error {
 		return trace.Wrap(err)
 	}
 
-	if len(alerts) == 0 {
+	if alerts == nil {
+		alerts = []types.ClusterAlert{}
```

Comment:
This is only necessary if format is `teleport.JSON`, so I would move it down under that case.

Reply:
I've moved this down here and in other examples.